data/xapleaux/xapleaux weapons.txt

```diff
@@ -0,0 +1,249 @@
+# Copyright (c) 2021 by MasterOfGrey
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+outfit "Small Trajectile Assembler"
+	category "Ammunition"
+	cost 2000000
+	thumbnail "outfit/sidewinder storage"
+	"mass" 12
+	"outfit space" 24
+	"assembly line" 1
+	description "The Small Trajectile Assembler has 1 assembly line space for a trajectile template to be loaded. A trajectile, which is much like a missile, is produced on demand by this machine from fuel with components and base resources contained in the template plug-in."
```

Comment:
So a missile is self-propelled or directed by remote control, and can change path.
A rocket is unguided, its trajectory is fixed once it is launched, controlled only by its launch conditions and total propellant.

These definitions matter for *us* because we pre-construct all our missiles or rockets, anything that uses them uses either one or the other.

A trajectile is simply something capable of trajection (following a trajectory) - i.e. a projectile, a missile, or a rocket. They are all types of trajectiles.

Since the specific type of thing is assembled here, not in advance, it's not any single one of them. It's capable of manufacturing any of them, and so it is generically a Trajectile Assembler.